pkg/assets/generated_assets.go

```diff
@@ -5359,9 +5359,9 @@ var vfsgenAssets = func() http.FileSystem {
 		"/terraform-modules/aws/flatcar-linux/kubernetes/versions.tf": &vfsgen۰CompressedFileInfo{
```

Comment:
To be honest, I'm not a fan of committing assets generation separately, as it breaks the atomicity of the commits. Maybe we can just commit everything as one commit?